docs/tutorials/revocation/revocation.collection.json

```diff
@@ -1,285 +1,313 @@
-              "    const responseJson = pm.response.json();",
-              "    pm.expect(responseJson.checks).to.eql([",
-              "        \"proof\",",
-              "        \"credentialStatus\"",
-              "    ]);",
-              "    pm.expect(responseJson.errors).to.eql([",
-              "        \"proof\",",
-              "        \"credentialStatus\"",
-              "    ]);",
-              "});"
-            ],
-            "type": "text/javascript"
-          }
-        }
-      ],
-      "request": {
-        "auth": {
-          "type": "bearer",
-          "bearer": [
-            {
-              "key": "token",
-              "value": "{{access_token}}",
-              "type": "string"
-            }
-          ]
-        },
-        "method": "POST",
-        "header": [],
-        "body": {
-          "mode": "raw",
-          "raw": "{\n    \"verifiableCredential\": {{revocable_credential}}\n}",
-          "options": {
-            "raw": {
-              "language": "json"
-            }
-          }
-        },
-        "url": {
-          "raw": "{{API_BASE_URL}}/credentials/verify",
-          "host": ["{{API_BASE_URL}}"],
-          "path": ["credentials", "verify"]
-        }
-      },
-      "response": []
-    }
-  ]
-}
+	"info": {
+		"_postman_id": "532f94ab-525e-4836-be30-5e857308f113",
+		"name": "Revocation",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "Get Access Token",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"exec": [
+							"pm.environment.set(\"access_token\", pm.response.json().access_token);"
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "POST",
+				"header": [
+					{
+						"key": "Accept",
+						"value": "application/json",
+						"type": "text"
+					}
+				],
+				"body": {
+					"mode": "raw",
+					"raw": "{\n    \"audience\": \"{{TOKEN_AUDIENCE}}\",\n    \"client_id\": \"{{CLIENT_ID}}\",\n    \"client_secret\": \"{{CLIENT_SECRET}}\",\n    \"grant_type\": \"client_credentials\"\n}",
+					"options": {
+						"raw": {
+							"language": "json"
+						}
+					}
+				},
+				"url": {
+					"raw": "{{TOKEN_ENDPOINT}}",
+					"host": [
+						"{{TOKEN_ENDPOINT}}"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "Get Organization DIDs",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"exec": [
+							"const { didDocument } = pm.response.json()",
+							"pm.environment.set(\"credential_issuer_id\", didDocument.alsoKnownAs[1]);",
+							""
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "GET",
+				"header": [
+					{
+						"key": "Accept",
+						"value": "application/json",
+						"type": "text"
+					}
+				],
+				"url": {
+					"raw": "{{API_BASE_URL}}/identifiers/{{ORGANIZATION_DID_WEB}}",
```

Comment:
This looks like it includes the identifiers change too?

Reply:
ah - good catch. I'll pull that for now, as it should be part of a different PR that updates this URL in all postman tests.